charts/ingress-nginx/Chart.yaml

```diff
@@ -24,4 +24,42 @@ annotations:
   # List of changes for the release in artifacthub.io
   # https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx?modal=changelog
   artifacthub.io/changes: |
-    - using helm-docs to populate values table in README.md
+    - #8120    Update go in runner and release v1.1.1
+    - #8119    Update to go v1.17.6
+    - #8118    Remove deprecated libraries, update other libs
+    - #8117    Fix codegen errors
+    - #8115    chart/ghaction: set the correct permission to have access to push a release
+    - #8098    generating SHA for CA only certs in backend_ssl.go + comparision of P…
+    - #8088    Fix Edit this page link to use main branch
+    - #8072    Expose GeoIP2 Continent code as variable
+    - #8061    docs(charts): using helm-docs for chart
+    - #8058    Bump github.com/spf13/cobra from 1.2.1 to 1.3.0
+    - #8054    Bump google.golang.org/grpc from 1.41.0 to 1.43.0
+    - #8051    align bug report with feature request regarding kind documentation
+    - #8046    Report expired certificates (#8045)
+    - #8044    remove G109 check till gosec resolves issues
+    - #8042    docs_multiple_instances_one_cluster_ticket_7543
+    - #8041    docs: fix typo'd executible name
+    - #8035    Comment busy owners
+    - #8029    Add stream-snippet as a ConfigMap and Annotation option
+    - #8023    fix nginx compilation flags
+    - #8021    Disable default modsecurity_rules_file if modsecurity-snippet is specified
+    - #8019    Revise main documentation page
+    - #8018    Preserve order of plugin invocation
+    - #8015    Add newline indenting to admission webhook annotations
+    - #8014    Add link to example error page manifest in docs
+    - #8009    Fix spelling in documentation and top-level files
+    - #8008    Add relabelings in controller-servicemonitor.yaml
+    - #8003    Minor improvements (formatting, consistency) in install guide
+    - #8001    fix: go-grpc Dockerfile
+    - #7999    images: use k8s-staging-test-infra/gcb-docker-gcloud
+    - #7996    doc: improvement
+    - #7983    Fix a couple of misspellings in the annotations documentation.
+    - #7979    allow set annotations for admission Jobs
+    - #7977    Add ssl_reject_handshake to defaul server
+    - #7975    add legacy version update v0.50.0 to main changelog
+    - #7972    updated service upstream definition
+    - #7964    Release updates for v1.1.0
+    - #7963    Change sanitization message from error to warning
+    - #7962    Start Release process for v1.1.0
+
```

Comment:
```suggestion
```

Reply:
Removed trailing line number 65

Reply:
removed trailing empty line and also 7962, 7963, 7964